requirements.txt

```diff
@@ -1,2 +1 @@
 # Astro Runtime includes the following pre-installed providers packages: https://docs.astronomer.io/astro/runtime-image-architecture#provider-packages
-apache-airflow-providers-cncf-kubernetes>=8.3.2
```

Comment:
Do we not need this because it is installed by default when running Airflow from the Astro Docker container?

Reply:
No it's because we already have the kubernetes package as part of observatory-platform. Having it here too seemed to result in some conflicts